music/songs/Wagon Wheel/Wagon Wheel.url

```diff
@@ -0,0 +1,2 @@
+youtube
```

Comment:
The first line of the .url file is the text to display in the table (the link you click).